api/controllers/SessionsController.js

```diff
@@ -5,52 +5,39 @@
  * @help        :: See https://sailsjs.com/docs/concepts/actions
  */
 
-const fs = require('fs');
-const path = require('path');
 const conditionHelper = require('../utils/ConditionHelpers');
 
 module.exports = {
-  index: function (req, res) {
-
-    let directory = path.resolve(__dirname, '../../sessions');
+  index: async function (req, res) {
+
+    const reports = await MedicalReport.findAll({limit: 10, attributes: ['id', 'firstName', 'lastName', 'socialInsuranceNumber']});
+    sails.log(`Query returned: ${JSON.stringify(reports)}`);
```

Comment:
This is going to get noisy ... can we set the level to silly, ie: `sails.log.silly` so you can still see them if you really want, but we're not filling up stdout?

Reply:
done